internal/install/recipe_installer.go

```diff
@@ -209,6 +210,10 @@ Our Data Privacy Notice: https://newrelic.com/termsandconditions/services-notice
 		return err
 	}
 
+	// Check if super-agent process is already running on host
+	SuperAgentProcessOnHost := i.processEvaluator.FindProcess(SuperAgentProcessName)
```

Comment:
Can this variable be lowercase? It's not being exported anywhere is it?

Reply:
Fixed it in bf1d274334373a1e4807055550fb0bccb8d41f2d and 999fb216954e7b924ce0777d78b1519cd7a0a320 (should've been in one commit).